index.html

```diff
@@ -0,0 +1,277 @@
+                <div class="info1">
+                    <div class="title11">
+                        <h2 class="title21">The E-Commerce </br>
+                            Pioneers Max 2023
+                        </h2>
+                    </div>
+                    <div class="squareInfo1">
+                        <h4 class="resumetitle1">
+                            The Global Summit brings together the community
+                            of experts, academics, and activists who comprise 
+                            the Creative Commons affiliate network in a 
+                            different country every two years. This year, 
+                            we hope to expand our invitation list including 
+                            organizations and individuals.
+                        </h4>
+                    </div>
+                    <div class="dateTitle1" id="dateTitle1">
+                        Please contact us per Email for nay further questions about CC Global Summit 2015!
+                    </div>
+                    <div class="siteDir1">
+                        ccsummit2015@anaxtasia.com
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="main11" id="main11">
+            <div class="firsttemplate11">
+                <div class="info1">
+                    <div class="ContainerSectionSpeakers">
+                        <div class="SectionSpeakT1">
+                            Creative Commons Global Summit 2015 Logo
+                        </div>
+                        <div class="RedLine"></div>
+                        <div class="dateTitle11">
+                            The logo of CC Global Summit 2015 was 
+                            decided through the logo competition 
+                            from 8, June to 7, July.
+                        </div>
+                        <img id="anaxtasia" src="./images/anaxtasia.jpg" alt="anax"/>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="main11" id="main11">
+            <div class="firsttemplate11">
+                <div class="info1">
+                    <div class="SectionSpeakT1">
+                        See the past CC Global Summits
+                    </div>
+                    <div class="RedLine"></div>
+                    <div class="penul" id="penul">a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</div>
+                    <div class="penul1">
+                        <div class="pict1">
+                            <img id="anaxtasia" src="./images/about_2011 1.png" alt="anax"/>
+                        </div>
+                        <div class="pict2">
+                            <img id="anaxtasia" src="./images/about_2013 1.png" alt="anax"/>
+                        </div>
+                    </div>
+                </div>
+            </div>
```

Comment:
- You are close, my friend! :muscle: 
- [ ] I am not sure that these highlighted sections should be here in the `Home` page. Kindly check Cindy's design [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and move it to the `About` page please.
- [ ] Also, I think you should watch the attached [video](https://watch.screencastify.com/v/UgXfOlXJAOKJFbwzGKif) if you haven't, please. It will save you of some troubles :pray:

Reply:
Hi @AbayomiOlaoye Thank you so much for your time. I'm doing the suggested changes for you and I found some differences with other code reviews that checked before. However. I made the changes suggested for you.

@brainconnect93 suggested reducing a size
![image](https://github.com/jovasal/Capstone/assets/35693330/4243271d-d5fb-457d-b58e-2164879d4ead)
Now I increase again and I reduce the images as suggested for you:
![image](https://github.com/jovasal/Capstone/assets/35693330/7a466af9-76e8-4d14-8f91-544e3ae2c193)

Reply:
Also, something was wrong when I saved and moved a button. Now is at the same level as you can see:

              
![image](https://github.com/jovasal/Capstone/assets/35693330/17dcd6dc-b369-4409-8879-4e503e178715)

Reply:
Margin changed:

before:
![image](https://github.com/jovasal/Capstone/assets/35693330/c0de1eb2-0863-481a-a54b-152343f068f9)
now:
![image](https://github.com/jovasal/Capstone/assets/35693330/433aab2b-8a26-4d50-85a9-665483e238cb)

Reply:
before:
![image](https://github.com/jovasal/Capstone/assets/35693330/9cc82847-1de4-412c-b346-83957b843d28)

now:
![image](https://github.com/jovasal/Capstone/assets/35693330/6200f43c-4233-4a6b-82ab-e34c8063fed1)

Reply:
![image](https://github.com/jovasal/Capstone/assets/35693330/0cf8c794-99c0-496e-bcbf-9f42212e0dcb)

Reply:
About, is on a different page, because About is like that:

![image](https://github.com/jovasal/Capstone/assets/35693330/acd255d7-fdd5-4e0a-a2a2-771c670166ea)


for that reason, about is only on one page:
![image](https://github.com/jovasal/Capstone/assets/35693330/180c12aa-c567-42ee-8f43-0d3d1b7933b3)

Reply:
On another page, I put design and @AbayomiOlaoye suggested, but no in About. is in a different page:
design is:
![image](https://github.com/jovasal/Capstone/assets/35693330/e58eaf68-5ff8-48ad-ba8b-21835fb5ab9c)
Now, my page is:
![image](https://github.com/jovasal/Capstone/assets/35693330/25c3764d-4d95-42f9-80f3-e22b213adf9c)

Reply:
this is my last extra review and the project are working fine. please, help me because is I do not pass, I can follow my program.

I am sure that my page is working fine. Also, buttons are matched with each part of the project as suggested @AbayomiOlaoye

Reply:
> * You are close, my friend! 💪
> * [ ]  I am not sure that these highlighted sections should be here in the `Home` page. Kindly check Cindy's design [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and move it to the `About` page please.
> * [ ]  Also, I think you should watch the attached [video](https://watch.screencastify.com/v/UgXfOlXJAOKJFbwzGKif) if you haven't, please. It will save you of some troubles 🙏

@jovasal, Kindly note that the code reviewer says here that the h

> * ot sure that these highlighted sections should be here in the `Home` page. Kindly check Cindy's design [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and move it to the `About` page please.

@jovasal Kindly read the statement of this reviewer saying that these highlighted sections should not be on the `Home` page, and Kindly check Cindy's design [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and move it to the `About` page, please.

You removed the highlighted sections from the home page, but you didn't move them to the about page